main.css

```diff
@@ -1,117 +1,142 @@
-html,
-body {
-    font-family: sans-serif;
-    margin: 0px;
-}
-
 /* header */
 
+html,body {
```

Comment:
is this still needed? normalize css should add it

Reply:
Yes. First I deleted it before the previous step (mobile) as you advised. 

But then smth happened and only using these tags helps me with deleting some white spaces in HTML :(

Reply:
this happened because you added normalize css in the wrong way [here](https://github.com/DmitriiTsy/Lyft_first/pull/2/files#diff-c7554cfe9adb429454a2b7f18f38c23598b1c6e1c2cd07cb98b6faaf5799048fR7) (you should use rel stylesheet instead of preload)